tensorflow/python/keras/layers/core.py

```diff
@@ -158,7 +163,7 @@ class Dropout(Layer):
    [ 5.    6.25]
    [ 7.5   8.75]
    [10.    0.  ]], shape=(5, 2), dtype=float32)
```

Comment:
We already have a usage example

Reply:
I added it because I thought it would be simpler to understand if an example with a sequential model was also used

Reply:
Yes, but the addition is not testable and is already included in all tutorials using Keras layers.

Reply:
Ok, fair enough. Should I close the PR?

Reply:
I think that's the good plan forward from here. Thank you for the contribution, let's try to get others